lib/AppInfo/Application.php

```diff
@@ -48,6 +49,10 @@ public function register(IRegistrationContext $context): void {
 			$context->registerReferenceProvider(ImageReferenceProvider::class);
 			$context->registerReferenceProvider(WhisperReferenceProvider::class);
 			$context->registerEventListener(RenderReferenceEvent::class, OpenAiReferenceListener::class);
+
+			if (version_compare($this->config->getSystemValueString('version', '0.0.0'), '27.0.0', '>=')) {
```

Comment:
Won't this, pre-v27, still load the STTProvider php file and then fail at the usage of the non-existing interface?

Reply:
I thought so too and tried and it works. I guess the STTProvider.php file is only loaded if it's actually used at run time.